packages/storybook/stories/code.stories.tsx

```diff
@@ -32,3 +32,40 @@ export const Default: Story = {
     children: `import { Code } from '@nl-design-system-candidate/code-react';`,
   },
 };
+
+export const Arabic: Story = {
+  name: 'Arabic (Qalb)',
+  args: {
+    children: `(قول "مرحبا يا عالم)`,
+  },
+  render: (args) => (
+    <div dir="rtl" lang="ar">
+      <Code>{args.children}</Code>
+    </div>
+  ),
+};
+
+export const Chinese: Story = {
+  name: 'Chinese (Traditional)',
+  args: {
+    children: `// 這是一個很長的中文程式碼註釋，可以延伸到第二行，而且還有一些很長的單字`,
+    lang: 'ch',
+  },
+};
+
+export const Japanese: Story = {
+  name: 'Japanese',
+
+  args: {
+    children: `// これは非常に長い日本語のコードコメントで、2行目にもなり、かなり長い単語も含まれています。`,
+    lang: 'jp',
+  },
+};
+
+export const German: Story = {
+  name: 'German',
+  args: {
+    children: `// Dies ist ein sehr langer deutscher Codekommentar, der in eine zweite Zeile übergehen könnte und auch einige ziemlich lange Wörter enthält`,
+    lang: 'de',
+  },
+};
```

Comment:
Ik denk dat deze stories opgedeeld moeten worden in 2 varianten:

- parent heeft een bepaalde taal (`lang="ar" dir="rtl"` bijvoorbeeld), de code heeft een andere taal (`lang="en"`)
- code heeft een specifieke taal (bijvoorbeeld: `lang="en"`), de story houdt geen rekening met een parent